modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/defs/BoxesEndpointDefs.scala

```diff
@@ -123,11 +123,11 @@ final class BoxesEndpointDefs[F[_]](settings: RequestsSettings) {
       .in(paging(settings.maxEntitiesPerRequest))
       .out(jsonBody[Items[OutputInfo]])
 
-  def getOutputsByAddressDef: Endpoint[(Address, Paging), ApiErr, Items[OutputInfo], Any] =
+  def getOutputsByAddressDef: Endpoint[(Address, Paging), ApiErr, OutputInfoM, Any] =
```

Comment:
We shouldn't break existing API format. Please add separate endpoint for that feature.

Reply:
done